src/game/music.h

```diff
@@ -35,12 +35,17 @@ void Music_Pause(void);
 // Unpauses the music.
 void Music_Unpause(void);
 
-// Returns currently playing track. If there is a track playing "over" a looped
-// track, returns the "overriding" track number.
-int16_t Music_CurrentTrack(void);
+// Returns the currently playing track. Ignores looped tracks.
+MUSIC_TRACK_ID Music_CurrentTrack(void);
+
+// Returns the last played track. Ignores looped tracks.
+MUSIC_TRACK_ID Music_LastPlayedTrack(void);
+
+// Returns the looped track.
+MUSIC_TRACK_ID Music_CurrentTrackLooped(void);
```

Comment:
Let's call this `Music_GetLoopedTrack` and rename this family of functions to `Music_Get…Track`. The looped track doesn't have to be currently playing.

Reply:
Ya good call. `Music_GetLastPlayedTrack` and `Music_GetCurrentLoopedTrack` aren't used atm, but I'm gonna leave them in there for future use.